setup.py

```diff
@@ -32,7 +32,6 @@
         'dask>=2.12',
         'ecmwf-api-client',
         'eofs',
-        'ESMPy',
```

Comment:
Now you are removing the dependency altogether? That seems drastic.

Reply:
gah :man_facepalming: Clearly not my best PR :grin: Let me put it back w/o pin